Assets/Scripts/ContainerButton.cs

```diff
@@ -0,0 +1,135 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ContainerButton : MonoBehaviour
+{
+    float selection_Container;
+    [Space(5)]
+    [Header("Heavy")]
+    public GameObject HeavySprite;
+    public GameObject HeavySelected;
+    [Space(5)]
+    [Header("Light")]
+    public GameObject LightSprite;
+    public GameObject LightSelected;
+
+    [Space(5)]
+    [Header("Melee")]
+    public GameObject MeleeSprite;
+    public GameObject MeleeSelected;
+
+    [Space(5)]
+    public GameObject HContainerBuy;
+    public GameObject LContainerBuy;
+    public GameObject MContainerBuy;
```

Comment:
as discussed ideally in the future this is changed to a list or array of gameobjects.